src/apps/common/submodules/portListing/portListing.js

```diff
@@ -75,8 +77,24 @@ define(function(require) {
 				isMonsterApp = _.isBoolean(args.isMonsterApp)
 					? args.isMonsterApp
 					: false,
+				parent,
```

Comment:
Please declare unassigned variables last.

Generally, this is helpful when later on you might need to assign a variable depending on one of the previous assigned variables.